astropy/units/tests/test_units.py

```diff
@@ -1013,33 +1013,6 @@ def test_enable_unit_groupings():
         assert imperial.inch in u.m.find_equivalent_units()
 
 
-def test_unit_summary_prefixes():
-    """
-    Test for a few units that the unit summary table correctly reports
-    whether or not that unit supports prefixes.
-
-    Regression test for https://github.com/astropy/astropy/issues/3835
-    """
-
-    from astropy.units import astrophys
-
-    for summary in utils._iter_unit_summary(astrophys.__dict__):
-        unit, _, _, _, prefixes = summary
-
-        if unit.name == "lyr":
-            assert prefixes
-        elif unit.name == "pc":
-            assert prefixes
-        elif unit.name == "barn":
-            assert prefixes
```

Comment:
`prefixes` is either `'Yes'` or `'No'` (since 1b9336329866887f437ebfca096b5ad5fac11ca2) and both are truthy, so it's been many years since these asserts have checked anything useful.